Simplify (#64)
component-driven/component-driven-development: src/exercises/4-Layout_Primitives/Readme.md

## 4.4 Stack primitive

> `Stack` is the layout primitive to create _stacking layouts_ i.e. when children evenly distributed vertically. Most designers don't think in CSS but rather in higher level abstractions like stacks, groups etc. Stack is a designer-friendly component that allows distribute its children vertically.

Reviewer: ```suggestion
`Stack` is the layout primitive to create _stacking layouts_ i.e. when children evenly distributed vertically. Most designers don't think in CSS but rather in higher-level abstractions like stacks or groups. Stack is a designer-friendly component that distributes its children vertically.
```

> In order to create custom props that are coupled with our design system, styled-system’s offers [system](https://styled-system.com/api#system) function.

Reviewer: Maybe?

```suggestion
To create a design system constrained custom prop, styled-system has the [system](https://styled-system.com/api#system) function.
```
